stories/MarkdownRegion.stories.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { MarkdownRegion } from '..';
+import { markdown-region } from '..';
```

Comment:
Thank you for your contribution. Actually I asked you to just do #19 because somebody else is already doing the others.

For this file here, there is no need to change.`..` means one directory up. It is implicit to be importing the file `../index.js`. Please read that file.

The component names should not change. What you changed here was the component name, not the filename.

Whatever is to the left of the word “from” is about the component name. Whatever is to the right of the word “from” is the file name.